File: backend/src/mock_data/events.py
from datetime import datetime

EVENTS = [
    {
        "id": 1,
        "title": "Introduction to FastAPI",
        "description": "Learn the basics of FastAPI.",
        "location": "Online",
        "start_time": datetime(2026, 8, 10, 18, 0),
        "end_time": datetime(2026, 8, 10, 20, 0),
        "created_at": datetime(2026, 7, 20, 12, 0),
        "event_category": {
            "id": 1,
            "title": "Workshop",
            "icon_name": "tools",
            "created_at": datetime(2026, 7, 1),
        },
        "topic_category": {
            "id": 2,
            "title": "Backend",
            "icon_name": "server",
            "created_at": datetime(2026, 7, 1),
        },
        "organizer": {
            "id": 1,
            "name": "Python Community",
            "type": "ORGANIZATION",
            "logo_url": None,
            "website_url": "https://example.org",
            "description": "Python enthusiasts",
            "created_at": datetime(2026, 7, 1),
        },
    }
]

File: frontend/src/services/mockData/events.ts
import { Event, EventList } from '@/types'

import { eventCategories } from './eventCategories'
import { topicCategories } from './topicCategories'
import { organizers } from './organizers'

export const events: Event[] = [
    {
        id: 1,
        title: "Introduction to React",
        description: "Learn the basics of React development.",
        location: "Siedziba KSI UJ",

        start_time: "2026-08-10T10:00:00Z",
        end_time: "2026-08-10T14:00:00Z",
        created_at: "2026-07-01T10:00:00Z",
        event_category: eventCategories[0],
        topic_categories: topicCategories,
        organizer: organizers[0]
    },
    {
        id: 2,
        title: "AI Workshop",
        description: "Hands-on workshop about artificial intelligence.",
        location: "Online",

        start_time: "2026-09-01T09:00:00Z",
        end_time: "2026-09-01T12:00:00Z",
        created_at: "2026-07-05T10:00:00Z",
        event_category: eventCategories[1],
        topic_categories: topicCategories,
        organizer: organizers[0]
    }
]

export const eventList: EventList = {
    id: 1,
    title: "Introduction to React",
    location: "Siedziba KSI UJ",
    start_time: "2026-08-10T10:00:00Z",
    end_time: "2026-08-10T14:00:00Z",
    created_at: "2026-07-01T10:00:00Z",
    event_category: eventCategories[0],
    topic_categories: topicCategories,
    organizer: organizers[0]
}
